### PR TITLE
Translate LayoutFooter and LayoutHeader

### PR DESCRIPTION
**Pull request by Zitle-Nancy**

### CODE REVIEW

File: content/community/nav.yml [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-- title: Tools
+      title: Recursos Externos
+- title: Herramientas
   items:
     - id: debugging-tools
-      title: Debugging
+      title: Depuración
     - id: component-workbenches
-      title: Component Workbenches
+      title: Herramientas de Trabajo para Component
```
Comment (icarlossz) [inline content/community/nav.yml]: `para componentes`, I think.

File: content/docs/nav.yml [hunk 2]
```diff
@@ -1,85 +1,85 @@
...
-      title: Accessibility
+      title: Accesibilidad
     - id: code-splitting
       title: Code-Splitting
     - id: context
-      title: Context
+      title: Contexto
     - id: error-boundaries
-      title: Error Boundaries
+      title: Limítes de Errores
```
Comment (icarlossz) [inline content/docs/nav.yml]: Should be `Límites` instead `Limítes`.

File: content/docs/nav.yml [hunk 3]
```diff
@@ -91,63 +91,63 @@
...
+        title: Hooks a Simple Vista
       - id: hooks-state
-        title: Using the State Hook
+        title: Usando el Hook de Estado
       - id: hooks-effect
-        title: Using the Effect Hook
+        title: Usando el Hook de Efecto
       - id: hooks-rules
-        title: Rules of Hooks
+        title: Reglas de Hooks
```
Comment (icarlossz) [inline content/docs/nav.yml]: I'd put `Reglas de los Hooks`.

File: content/docs/nav.yml [hunk 4]
```diff
@@ -91,63 +91,63 @@
...
       - id: hooks-custom
-        title: Building Your Own Hooks
+        title: Construyendo tus Propios Hooks
       - id: hooks-reference
-        title: Hooks API Reference
+        title: Referencia de API de Hooks
```
Comment (icarlossz) [inline content/docs/nav.yml]: I'd put `Referencia al API de los Hooks`.

**Comment by Zitle-Nancy** [comment]
You're welcome, It was a pleasure , if you need anything else just let me know.

**Pull request merged**